DiscordRPC/Configuration.cs

```diff
@@ -24,9 +24,9 @@ public class Configuration
 		public string CdnHost { get; set; }
 
 		/// <summary>
-		/// The type of enviroment the connection on. Usually Production. 
+		/// The type of environment the connection on. Usually Production. 
 		/// </summary>
-		[JsonProperty("enviroment")]
-		public string Enviroment { get; set; }
```

Comment:
Should we keep this property and make it point to Enviro**n**ment? Then add a `[System.Obsolete]` on top to indicate it has been renamed?  Doing it like this will allow this to be a simple patch instead of a minor of major version.

Reply:
Done! Good suggestion :+1: